apps/emqx_management/src/emqx_mgmt_api.erl

```diff
@@ -624,8 +581,9 @@ to_timestamp(I) when is_integer(I) ->
 to_timestamp(B) when is_binary(B) ->
     binary_to_integer(B).
 
-aton(B) when is_binary(B) ->
-    list_to_tuple([binary_to_integer(T) || T <- re:split(B, "[.]")]).
+to_ip(IP0) when is_binary(IP0) ->
+    {ok, IP} = inet:parse_address(binary_to_list(IP0)),
+    IP.
```

Comment:
Just a minor observation: this will now successfully accept IPv6 as inputs, and it previously didn't.

Probably not an issue, more like an emergent feature.  :smile_cat:

Reply:
Oh right, even more important to be consistent with `ip_port` then.